docs/articles/documentation/using-testcafe/command-line-interface.md

```diff
@@ -268,12 +269,23 @@ Note that only one reporter can write to `stdout`. All other reporters must outp
 
 ### -s \<path\>, --screenshots \<path\>
 
-Enables screenshot capturing and specifies the directory where screenshots are saved.
+Enables screenshot capturing and specifies the root directory where screenshots are saved.
 
 ```sh
 testcafe all tests/sample-fixture.js -s screenshots
 ```
 
+#### Path Patterns
+
+The captured screenshots are organized into subdirectories within the root directory. The screenshots' relative path and name are defined using the default screenshot path patterns:
```

Comment:
The screenshots' relative path and name are defined using the default screenshot path patterns: -> The default screenshot path patterns are used to define screenshots' relative path and name: